.github/workflows/lint.yml

```diff
@@ -4,8 +4,15 @@ on:
   pull_request:
     branches: [master, main]
 
+permissions:
+  contents: read # to fetch code (actions/checkout)
+
 jobs:
   super-linter:
+    permissions:
+      contents: read # to fetch code (actions/checkout)
+      checks: write # to mark the status of each individual linter run
```

Comment:
Phpstan also needs write permissions

Reply:
I have added `checks: write` according [to the comment](https://github.com/github/super-linter/blob/d47cecf31ccb100f515b9c729be24ef3f7c501b8/README.md?plain=1#L124) in super-linter action. I think phpstan is not an action and doesn't know how to call github api to make use of the permission, although GitHub should show the overall check for the job as a whole - failed or succeed. Please, let me know if you still think it is better to grant `checks: write` to the phpstan job.